components/Hero/Hero.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Button } from "../Button/Button";
+import Image from "next/image";
+
+export interface HeroProps extends React.HTMLProps<HTMLElement> {
+  title: string;
+  subTitle: string;
+  image: {
+    src: string;
+    alt: string;
+  };
+}
+
+export const Hero = ({ title, subTitle, image }: HeroProps) => {
+  return (
+    <section className="relative py-10">
+      <div className="flex max-w-screen-xl flex-col lg:ml-auto lg:flex-row lg:items-center">
+        <div className="mb-14 mr-2 lg:w-6/12">
+          <h1 className="max-w-4xl text-blue">{title}</h1>
+          <p className="my-4 mb-9 font-inter text-xl font-medium text-blue">
```

Comment:
Yea, it would be better to have `mb` for text tags. Also, I noticed, that `Button` component in design has 3px rounded corners, but in code it isn't. And className prop don't work. It's intended?

Reply:
The differences between the design and the component wasn't intended. Now we are trying to fix those where is needed.